src/applications/search/reducers/index.js

```diff
@@ -41,6 +43,7 @@ function SearchReducer(state = initialState, action) {
         currentPage,
         perPage,
         totalPages,
+        searchesPerformed,
```

Comment:
I added this counter into the reducer so we could easily tell when it's the first results set and therefore not to set browser focus.